### PR TITLE
Testing NW

### PR DESCRIPTION
**Pull request by benkeen**
The CI was all green. A blank PR should also be green. Confirming.

### CODE REVIEW

**Comment by benkeen** [comment]
The failing CI is due to a recent couch change. Adding in a `--nodes=1` param fixes the problem, see second commit.

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Fauxton
 
-
 Fauxton is the new Web UI for CouchDB. To get it running in development on your machine. Follow the steps below.
```
Comment (michellephung) [inline readme.md]: :pineapple:

**Comment by michellephung** [comment]
+1

**Comment by benkeen** [comment]
This should be moot now, @rnewson fixed the issue. Going to re-run one of the other PRs to confirm then close this sucker.

**Comment by benkeen** [comment]
Closing.

**Pull request closed without merging**